draft-irtf-panrg-path-properties.md

```diff
@@ -311,16 +311,18 @@ Metrics such as loss patterns {{RFC3357}} and loss episodes {{RFC6534}} can be e
 # Security Considerations
 
 If entities are basing policy or path selection decisions on path properties, they need to rely on the accuracy of path properties that other devices communicate to them.
-In order to be able to trust such path properties, entities may need to establish a trust relationship or be able to verify the authenticity, integrity, and correctness of path properties received from another entity.
+In order to be able to trust such path properties, entities may need to establish a trust relationship or be able to independently verify the authenticity, integrity, and correctness of path properties received from another entity.
 
 Entities that reveal their target path properties to the network can negatively impact their own privacy, e.g., if the target property leaks personal information about a user, such as their identity or which (type of) application is used.
 Such information could then allow network operators to block or re-prioritize traffic for certain users and/or application.
 Conversely, if privacy enhancing technologies, e.g., MASQUE proxies [RFC9298], are used on a path, the path may only be partially visible to any single entity.
 This may diminish the usefulness of path-aware technologies over this path.
-Security related properties such as confidentiality and integrity protection of payloads are difficult to characterize since they are only meaningful with respect to a threat model which depends on the use case, application, environment, and other factors.
+
+The need for, and potential definition of, security and privacy related path properties, such as confidentiality and integrity protection of payloads, are the subject of ongoing discussion and research, e.g., see {{?RFC9049}} and {{?RFC9217}}. As the thinking around such properties is not mature enough, such properties are out of scope for this document.
+One aspect discussed in this context is that security related properties are difficult to characterize since they are only meaningful with respect to a threat model which depends on the use case, application, environment, and other factors.
 Likewise, properties for trust relations between entities cannot be meaningfully defined without a concrete threat model, and defining a threat model is out of scope for this draft.
-Properties related to confidentiality, integrity, and trust are orthogonal to the path terminology and path properties defined in this document.
-Such properties are tied to the communicating nodes and the protocols they use (e.g., client and server using HTTPS, or client and remote network node using VPN) while the path is typically oblivious to them.
+In addition, we suspect that properties related to confidentiality, integrity, and trust are orthogonal to the path terminology and path properties defined in this document.
```

Comment:
I'm not too happy about the word "suspect", but I can't come up with a more fitting word.
Maybe we can combine it with the next sentence and rephrase it to: "Properties related to confidentiality, integrity, and trust seem to be orthogonal to the path terminology and path properties defined in this document, since they are tied ..."
What do you think?

Reply:
That sounds good to me, I'll apply your change.